assets/js/modules/search-console/components/dashboard-details/DashboardDetailsWidgetKeywordsTable.js

```diff
@@ -29,19 +29,21 @@ import Data from 'googlesitekit-data';
 import LegacySearchConsoleDashboardWidgetKeywordTable from '../dashboard/LegacySearchConsoleDashboardWidgetKeywordTable';
 import DashboardModuleHeader from '../../../../components/dashboard/DashboardModuleHeader';
 import Layout from '../../../../components/layout/Layout';
-import { STORE_NAME } from '../../datastore/constants';
+import { STORE_NAME, DATE_RANGE_OFFSET } from '../../datastore/constants';
 import { STORE_NAME as CORE_SITE } from '../../../../googlesitekit/datastore/site/constants';
-import { getCurrentDateRangeDayCount } from '../../../../util/date-range';
+import { STORE_NAME as CORE_USER } from '../../../../googlesitekit/datastore/user/constants';
```

Comment:
Now that #2563 is merged these imports can be cleaned up and are the likely source of merge conflicts in the PR—this can now be imported directly as `CORE_USER` instead of `STORE_NAME as CORE_USER`.